content/usage/usage.md

```diff
@@ -7,7 +7,7 @@ chapter: true
 
 #### If **Reacli** is installed globally
 
-If you have installed **Reacli** globally (see [installation]({{< relref "install.md" >}})), you can directly use **Reacli** like:
+If you have installed **Reacli** globally (see [installation]({{< relref "install.md" >}})), you can directly use it like this:
```

Comment:
My point was than "If you have installed **Reacli** globally" is not proper English...

Reply:
Sorry I completely missed it 😞

Reply:
b79ec58